.github/workflows/checks.yml

```diff
@@ -23,6 +23,8 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
This adds about 18s to the run, not that happy about that. Looks like it's needed to do a diff against base though?